packages/node/build-and-push-to-ecr.sh

```
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
+
+set -euxo pipefail
+
+docker build \
+  -t codex-node:latest \
+  -t 523044037273.dkr.ecr.us-east-2.amazonaws.com/codex-node:latest \
+  -f Dockerfile \
+  $(git rev-parse --show-toplevel)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)